Source/Core/Core/IOS/ES/TitleManagement.cpp

```diff
@@ -696,20 +707,9 @@ IPCCommandResult ES::ExportContentData(Context& context, const IOCtlVRequest& re
 
 ReturnCode ES::ExportContentEnd(Context& context, u32 content_fd)
 {
-  const auto iterator = context.title_export.contents.find(content_fd);
-  if (!context.title_export.valid || iterator == context.title_export.contents.end() ||
-      iterator->second.content.m_position != iterator->second.content.m_content.size)
-  {
+  if (!context.title_import.valid || !context.title_import.content.valid)
```

Comment:
This no longer seems to error if the full content wasn't read (unless I missed something in `CloseContent`, which is highly likely). Is that intentional?